pkg/volume/plugins.go

```diff
@@ -411,12 +411,13 @@ func NewPersistentVolumeRecyclerPodTemplate() *api.Pod {
 					VolumeSource: api.VolumeSource{},
 				},
 			},
+
 			Containers: []api.Container{
 				{
 					Name:    "pv-recycler",
 					Image:   "gcr.io/google_containers/busybox",
 					Command: []string{"/bin/sh"},
-					Args:    []string{"-c", "test -e /scrub && echo $(date) > /scrub/trash.txt && find /scrub -mindepth 1 -maxdepth 1 -delete && test -z \"$(ls -A /scrub)\" || exit 1"},
+					Args:    []string{"-c", "test -e /scrub && rm -rf /scrub/..?* /scrub/.[!.]* /scrub/* ; echo $(date) > /scrub/trash.txt && exit 0 "},
```

Comment:
Probably want a separate PR for this change, since @thockin was backporting this fix to the older 1.1.x branch.  Can't do that with the fix being in this PR.